src/main/io/osd.c

```diff
@@ -668,6 +671,20 @@ static bool osdDrawSingleElement(uint8_t item)
                 break;
             }
 
+#ifdef USE_ESC_SENSOR
+            // Show warning if we lose motor output
+            if (enabledWarnings & OSD_WARNING_ESC_FAIL && ARMING_FLAG(ARMED)) {
+                char escErrMsg[OSD_FORMAT_MESSAGE_BUFFER_SIZE] = "ESC";
+                char *escIds = escErrMsg + 3;
+                for (int i = 0; i < getMotorCount(); i++) {
+                    escSensorData_t *escDatai = getEscSensorData(i);
+                    escIds[i] = escDatai->rpm <= 0 ? '0' + i + 1 : ' ';
```

Comment:
Does anyone have a better suggestion for a lower bound for the minimum viable rpm? While 0 or less is certainly indicative of a problem, so would be 1. I imagine anything significantly less than idle should be warned. For ref, my idle reports around 90. Is it worth increasing from 0 to 10 or maybe 50?

Reply:
Not sure if anything more than 0 is desirable - when flying 3D the reported rpm will probably go close to 0 when inverting.

Reply:
I guess if it becomes necessary we can have a configurable threshold and check if abs(rpm) <= threshold.